### DISCUSSION TITLE
Some help on download setting.

### DISCUSSION THREAD

**Discussion by Cren0** [TumblThreeApp/TumblThree#325] [Q&A (Question & Answer)]
so I´ve been using this app for a while now, and it´s working great for me. Some issues I still need some help.
1) in Tumblr there seems to be 'webp' files, how can I save these files (what settings I need to put on) ?
2) I searched info about the issue: blog is hidden, is there some workaround to help getting those blogs downloaded also (tried setting the one value to 1 but no help there)

**Comment by thomas694** [comment]
It's great to hear that you like TumblThree. Give it a star.

1. I don't know many blogs that have webp photos. If they are uploaded and used in the official photo structures they should be downloaded without problems. Embedded images (hosted on other platforms) can be downloaded by activating "search for image patterns".
If you have an appropriate/safe blog whose Webp aren't downloaded you can post it (or email), maybe we can find and fix something.

2. Hidden blogs should be handled automatically. Normally you can see the difference by type "Tumblr Hidden" and the file extension "tmblrpriv".

**Comment by Cren0** [comment]
Ok, now all seems to work just fine, hidden blogs also download perfectly, needed a log in, also needed to make some adjustments to settings. Thank you very much !
